common/libs/validators.ts

```diff
@@ -86,9 +86,15 @@ function validateEtherAddress(address: string): boolean {
 
 export function isValidPrivKey(privkey: string | Buffer): boolean {
```

Comment:
EthereumJS lib that handles this? Not sure how much I like having this custom logic as part of our app code, and definitely not additional regex.